packages/s/seafile/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : seafile
-version    : 8.0.7
-release    : 29
+version    : 9.0.4
+release    : 30
 source     :
-    - https://github.com/haiwen/seafile/archive/refs/tags/v8.0.7.tar.gz : c6c703476907b9c84ec25fd681d85bced876ee2528409060f7e24058c73cad0a
+    - https://github.com/haiwen/seafile/archive/refs/tags/v9.0.4.tar.gz : 6374569f05eb6f8d1632fbfc4356754078308e2a0605aed4e798219b4eafbb51
+homepage   : https://www.seafile.com/
```

Comment:
You need to rebuild this package locally and amend your commit. You added the `homepage` key after you built the package and the key is not in the pspec file.

Reply:
Damn, my bad. Do you want me to amend and force push or are you squashing the commits anyway?

Reply:
Amend and force push